app/src/app/components/view-profile/view-profile.component.html

```diff
@@ -56,20 +56,28 @@ <h4 class="app-profile-general-dropdown-heading">{{'APP.EDIT_PROFILE.TELEPHONE'
           <p class="app-profile-general-dropdown-details" *ngIf="generalSectionButtonsToggle">{{personDetails?.telephone}}</p>
           <input type="tel" name="Telephone" value="" formControlName="telephone" *ngIf="!generalSectionButtonsToggle"
             class="app-profile-general-dropdown-input-details app-selected">
+          <mat-error class="app-error-message" *ngIf="profileForm.get('telephone').invalid  &&
+            (profileForm.get('telephone').dirty || profileForm.get('telephone').touched)">
+            {{ 'APP.EDIT_PROFILE.INVALID_INPUT' | translate }}
+          </mat-error>
         </div>
         <mat-divider class="app-general-dropdown-divider"></mat-divider>
         <div class="app-general-dropdown-details">
           <h4 class="app-profile-general-dropdown-heading">{{'APP.EDIT_PROFILE.MOBILE' | translate}}</h4>
           <p class="app-profile-general-dropdown-details" *ngIf="generalSectionButtonsToggle">{{personDetails?.mobile}}</p>
         <input type="tel" value="" formControlName="mobile" *ngIf="!generalSectionButtonsToggle"
           class="app-profile-general-dropdown-input-details app-selected">
+        <mat-error class="app-error-message" *ngIf="profileForm.get('mobile').invalid  &&
+          (profileForm.get('mobile').dirty || profileForm.get('mobile').touched)">
+          {{ 'APP.EDIT_PROFILE.INVALID_INPUT' | translate }}
+        </mat-error>
         </div>
       </div>
     </div>
   </div>
 
-  <div class="app-profile-login-row" [formGroup]="profileForm">
-    <div class="app-profile-login" >
+  <div class="app-profile-login-row" [formGroup]="profileForm" hidden>
```

Comment:
why it's hidden?

Reply:
This password change section is in development and probably will move to next release train. So made it hidden till this functionality is created.

Reply:
I suggest moving this functionality into a separate PR and merging when it's needed, instead of putting invisible components with this PR

Reply:
sure